pinns/src/pinns.c

```diff
@@ -54,9 +56,10 @@ int main(int argc, char **argv) {
       {"filename", required_argument, NULL, 'f'},
       {"uid-mapping", optional_argument, NULL, UID_MAPPING},
       {"gid-mapping", optional_argument, NULL, GID_MAPPING},
+      {"sysctl", optional_argument, NULL, 's'},
```

Comment:
looks like `s/optional_argument/required_argument/`.

Reply:
no it should be optional. below, I check whether sysctls was specified before processing them